recipes/pandoc/notes.md

```diff
@@ -0,0 +1,3 @@
+* The Linux package doesn't work on systems older than CentOS 6
+* The OS X package works on 10.7+
+* The Windows package works on 32 and 64 bit archs
```

Comment:
It's in the recipe directory, but if that should be in any way user visible, it should be in the meta.yml...

But: IMO it's pretty much not interesting for users? 10.7 was released in 2011, windows is simple just states "it works everywhere" and Centos6 was also released in 2011...